lib/internal/Magento/Framework/Filter/LocalizedToNormalized.php

```diff
@@ -60,14 +61,16 @@ public function setOptions(?array $options = null)
      *
      * Normalizes the given input
      *
-     * @param  string $value Value to normalized
+     * @param string $value Value to normalized
      * @return string|array The normalized value
      */
     public function filter($value)
     {
         if (is_numeric($value)) {
-            $numberParse = new NumberParse($this->_options['locale'], NumberFormatter::PATTERN_DECIMAL);
-            return (string) $numberParse->filter($value);
+            $numberParse = new NumberParse(
```

Comment:
Removing `NumberFormatter::PATTERN_DECIMAL` as a second parameter from here might introduce backward compatibility issues. Please consider using an alternative approach to handle this.